blocksuite/affine/shared/src/__tests__/helpers/affine-template.unit.spec.ts

```diff
@@ -80,4 +81,79 @@ describe('helpers/affine-template', () => {
       `;
     }).toThrow();
   });
+
+  it('should handle text selection with anchor and focus', () => {
+    const host = affine`
```

Comment:
Do you plan to replace it with tsx in this pr or next pr?

Reply:
Yeah, it's a future project(RFC --> Test Framework).